thoth/storages/graph/postgres.py

```diff
@@ -2799,6 +2799,7 @@ def get_hardware_environments_all(
     def get_software_environments_all(
         self,
         is_external: bool = False,
+        include_ids: bool = False,
```

Comment:
Do we use this query elsewhere which requires keeping the API strictly backward compatible?

Reply:
No to my knowledge 🤔 should we make it default choice?

Reply:
Maybe metrics-exporter

Reply:
I did find it used in metrics-exporter. I think we can safely just extend the dictionary returned with additional values introduced - no need to add `include_ids` parameter.

Reply:
> Maybe metrics-exporter

ok no, we are good

Reply:
> I did find it used in metrics-exporter. I think we can safely just extend the dictionary returned with additional values introduced - no need to add `include_ids` parameter.

Adjusted, thanks!!